internal/onboarding/ms365_test.go

```diff
@@ -207,7 +196,7 @@ resource "mondoo_integration_ms365" "this" {
     pem_file = join("\n", [tls_self_signed_cert.credential.cert_pem, tls_private_key.credential.private_key_pem])
   }
   depends_on = [azuread_service_principal.mondoo, azuread_application_certificate.mondoo, azuread_directory_role_assignment.global_reader]
-  name       = "subscription-"
+  name       = "subscription-88afcea5fb91"
```

Comment:
Do we have a different prefix for this?
Now that this is no longer linked to a subscription, we might want to avoid the work here.

Reply:
Good call, I will update it in a diff PR. Thanks @czunker